api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -838,3 +838,11 @@ message RequestIDExtension {
   // Request ID extension specific configuration.
   google.protobuf.Any typed_config = 1;
 }
+
+message HttpRequestHeaderMatchInput {
+  string header_name = 1;
+}
+
+message HttpResponseHeaderMatchInput {
+  string header_name = 1;
+}
```

Comment:
Not sure if these should be tied to the HCM config or standalone?

Reply:
I'm confused about what these are supposed to be :) Can you add some docs?

Reply:
Will review tomorrow but +1 please more docs to help review.

/wait